### ISSUE TITLE
"Redirect to primary URL" feature in Subtext is incompatible with new staging environment

### ISSUE THREAD

**Issue by jeremy-jameson** [technology-toolbox/Subtext#3]
Subtext currently supports a "primary" URL for each blog as well as one or more "alternate" URLs. The primary URL is specified via the **Host** and **Application** columns in **subtext_Config**. Alternate URLs are specified in the **subtext_DomainAlias** table (using similar **Host** and **Application** columns).

When an HTTP request is received for an alternate URL (i.e. it does _not_ match the **Host** or **Application** specified in **subtext_Config** but _does_ match a row in the **subtext_DomainAlias** table), the application redirects to the primary URL for the blog.

This redirect feature has been used in the past to support URLs such as the following:

- https://www.technologytoolbox.com/blog/jjameson
- https://technologytoolbox.com/blog/jjameson

In other words, browsing to https://technologytoolbox.com/blog/jjameson redirects to https://www.technologytoolbox.com/blog/jjameson.

However, this feature is incompatible with the new **staging** deployment slot (which shares the same database instance as production). When browsing to the staging environment, the "redirect to primary URL" feature results in the content being served from the production environment instead.

To support the **staging** deployment slot, Subtext needs to be updated to support alternate URLs _without_ changing the original hostname specified in the request.
